pipeline_dp/dp_computations.py

```diff
@@ -38,13 +38,20 @@ def linf_sensitivity(self, metric):
         if metric == pipeline_dp.Metrics.MEAN:
             return self.max_contributions_per_partition * abs(
                 self.middle() - self.low)
+        if metric == pipeline_dp.Metrics.VAR:
+            return self.max_contributions_per_partition * abs(
+                self.middle_squares() - self.low ** 2)
         # TODO: add value for variance
         raise ValueError("Invalid metric")
 
     def middle(self):
         """"Returns the middle point of the interval [low, high]."""
         return self.low + (self.high - self.low) / 2
 
+    def middle_squares(self):
+        """"Returns the middle point of the interval [low^2, high^2]."""
+        return self.low ** 2 + (self.high ** 2 - self.low ** 2) / 2
```

Comment:
This is incorrect. As an example, imagine low = -5, high = 5. This will return 25 + (25 - 25) / 2 = 25. But actually, the minimum squared value is 0 (0 ** 2), and the maximum squared value is 25 (5 ** 2 or -5 **2), so the correct midpoint is 12.5.